yatsm/algorithms/ccdc.py

```diff
@@ -220,6 +220,9 @@ def fit(self, X, Y, dates):
 
             while not self.monitoring and self.can_monitor:
                 self.train()
+                # Ensure all bands in Y are fit in case we can't monitor
+                # after training
+                self._update_model()
                 self.here += 1
 
```

Comment:
Got it! Makes more sense now. I can modify this later. Should we close the pull request and make a new, clean one?